pom/pom.xml

```diff
@@ -227,11 +226,6 @@
                 <plugin>
```

Comment:
This plugin (`maven-compiler-plugin`) is already global defined in `build / plugins` and you don't need to add this again.
See [Line 465](https://github.com/Viascom/groundwork/blob/7fd66d0b3680915a2f7daee16058e32d96a202b8/pom/pom.xml#L465)